docs/_documentation/configuration.md

```diff
@@ -21,6 +21,14 @@ basic usage.
 }
 ~~~~
 
+## Portable Mode {#portableMode}
```

Comment:
I had no idea the `{#foo}` thing worked. I was using a js-shim. I should go back and fix that [: